docs/common/StringResource.md

```diff
@@ -0,0 +1,35 @@
+# Localized strings
+
+All strings shown to the user must be localized. The `StringResource` service provides an easy way to access those strings.
+
+## Usage
+
+### Creating a localized string
+
+The project you are working in should have a directory under `Strings\en-us`, containing a file called `Resources.resw`.
```

Comment:
Should this include instructions on making a resource file that isn't the `en-us` language?

Reply:
No -- we create all our strings in English and this is specifically for Dev Home, not localization in general.